client/src/pages/ProfileSetting.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import Onboarding from "../components/Onboarding";
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography'; 
+import TextField from '@material-ui/core/TextField';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import DropdownSelect from '../components/DropdownSelect';
+
+const moment = require('moment-timezone');
+// const abbrs = moment.tz.zonesForCountry('CA').map((z) => moment.tz(z).zoneAbbr());
+
+// Selected timezones only for Canada, otherwise there would be too many
+const timezones = moment.tz.zonesForCountry('CA').map((z) => {
```

Comment:
@sina-jamshidi I used actual timzones i.e. america/toronto instead of abbreviations because it would be more accurate (see screenshot below). Let me know if that is ok.

Reply:
Yup that's great! I'll look at the PR itself later